src/components/NavBar/style.css.ts

```diff
@@ -38,7 +36,6 @@ export const leftSideContainer = style([
 
 export const searchContainer = style([
```

Comment:
The searchbar is no longer centered in ENG or JPN
![Screen Shot 2023-05-16 at 10 32 27 ](https://github.com/Uniswap/interface/assets/11512321/fbf1ab46-8261-4bd9-9fa0-9bb07c3f9bb5)
![Screen Shot 2023-05-16 at 10 33 34 ](https://github.com/Uniswap/interface/assets/11512321/5c938a57-55de-44ae-9581-c78f3fb1c351)

Reply:
Compared to current production:
![Screen Shot 2023-05-16 at 10 37 22 ](https://github.com/Uniswap/interface/assets/11512321/ea55ac03-a6ee-4580-b188-8afa3fa998f0)

Reply:
@cbachmeier If the search is always centered then it will cause the left side to be squeezed, which is the case now. So I used `space-between` to balance the space between the left and the right.

Reply:
As a design choice, we will always want the searchbar to be centered on the swap modal and instead would prefer the text in the navbar to wrap.